tests/installation/sle11_releasenotes.pm

```diff
@@ -10,11 +10,19 @@ sub run(){
     if (get_var("ADDONS")) {
         if (check_screen 'release-notes-tab') {
             foreach $a (split(/,/, get_var('ADDONS'))) {
-                send_key 'alt-u';
-                assert_screen "release-notes-$a";
+                if (check_var('ADDONS', 'smt')) {
```

Comment:
now that's not *exactly* what I meant :)

the loop is splitting $ADDONS and as such within the loop you want to check $a not $ADDONS